blogposts/announcing-sourcegraph-3.3.md

```diff
@@ -130,35 +117,60 @@ The initial version of the extension provides support for the most popular langu
 
-- [**Bitbucket server fixes**](https://github.com/sourcegraph/sourcegraph/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+label%3Abitbucket+milestone%3A3.3+)<br />
-  Many UI improvements so Sourcegraph feels like a native integration of Bitbucket Server.<br /><br />
-- [**Browser extension fixes**](https://github.com/sourcegraph/sourcegraph/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+milestone%3A3.3+label%3Abrowser-extension+)<br />
-  Better Sourcegraph connection error handling and prompts for when the extension must be manually activated, e.g. if using GitLab.com, as well as new [browser extension troubleshooting tips](https://docs.sourcegraph.com/integration/browser_extension#troubleshooting).
+## Grant permissions on your code host
 
-As part of making the browser extension more robust, we're increasing end-to-end test coverage to catch code host related issues such as GitHub DOM changes faster. This reduces the time from issue logged, to patched and released, the goal being to catch any code host integration issues before our users do.
+Granting permissions in the Sourcegraph browser extension is necessary when using anything except GitHub.com. It's now easier to discover:
 
-<div id="code-intel"></div>
+<div style="text-align: center">
 
-## Code intel: Improved Swift support and directory/file scoped symbols
+  ![](/blog/sourcegraph-browser-extension-enable-prompt.png)
+  
+</div>
 
-Swift code intelligence is more comprehensive now that symbol detection has been improved, and the symbols sidebar is now scoped to the current file/directory you’re viewing:
+## "Go to definition" using dependency graph
 
-![](/blog/swift.png)
+Definitions are now filtered using `import` / `require` statements to reduce false matches and improve accuracy.
 
-Other updates include:
+![](/blog/filtered-imports.png)
 
-- Jump-to-definition is more precise now that the definitions are filtered by the import statements at the top of the file.
-- The Go language server responds faster upon first visiting a page now that it caches zip archives more intelligently.
+## Swift code navigation
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/331859021?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/331859021" target="_blank">View on Vimeo</a></p>
+</p>
+
+See it in action on [Sourcegraph.com](https://sourcegraph.com/github.com/googleprojectzero/fuzzilli/-/blob/Sources/Fuzzilli/Configuration.swift#L64:27) or enable it for all users on your private instance by adding it to the `extensions` object in **Site admin > global settings** (path is `/site-admin/global-settings`):
+
+```json
+..."extensions": {
+    ...
+    "sourcegraph/swift": true
```

Comment:
Delete this JSON example now that I added the link to https://docs.sourcegraph.com/admin/extensions#enable-an-extension-for-all-users above